intake_esm/core.py

```diff
@@ -7,7 +7,7 @@
 import pandas as pd
 import pydantic
 import xarray as xr
-import xcollection as xc
+from datatree import DataTree
```

Comment:
could we make datatree a soft/optional dependency?

Reply:
this would require importing it inside the `.to_datatree()` method and throwing an error when this method is called without `xarray-datatree`...

Reply:
Yup! I can go ahead and add that